src/data/reducers/auth.js

```diff
@@ -14,21 +14,29 @@ const initialState = {
 
 export default function (state = initialState, action) {
   const { type, payload } = action;
+  let firstName, lastName, email, userId, teamId;
```

Comment:
Nit: would it make sense to define these constants inside of each case? 

Ex. `const { firstName, lastName, email, userId, teamId } = payload;`

Reply:
Yeah I agree and I tried that, but I was getting an error that those constants were already defined in the first switch case, so I ended up going with the let statement (not sure if there's a better way). I just took this out and went with your approach in the next comment though, so this won't be a problem. Thanks!